go.mod

```diff
@@ -10,6 +10,7 @@ require (
 )
 
 require (
+	github.com/cli/browser v1.2.0 // indirect
```

Comment:
This is a direct dependency. You need to run `go mod tidy` to clean up dependencies in the go.mod / go.sum